terraform/aws/jenkins/vpc.tf

```diff
@@ -0,0 +1,107 @@
+resource "aws_vpc" "vpc" {
+  cidr_block = var.cidr_block
+  tags = {
+    Name = "jenkins_vpc"
+  }
+}
+resource "aws_subnet" "subnet" {
+  vpc_id                  = aws_vpc.vpc.id
+  cidr_block              = var.subnet
+  availability_zone       = "us-east-2b"
```

Comment:
We should not hard code the region, but use the region that the cluster is deployed into.  The 'TF_VAR_AWS_REGION' exist for Terraform to pick up from the docker-compose file!